packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -4,12 +4,12 @@ import 'dart:html' as html;
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'package:meta/meta.dart';
 import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
-import 'src/navigator.dart' as navigator;
 
 /// The web implementation of [UrlLauncherPlatform].
 ///
 /// This class implements the `package:url_launcher` functionality for the web.
 class UrlLauncherPlugin extends UrlLauncherPlatform {
+  static final _iosPlatforms = ['iPhone', 'iPad', 'iPod'];
```

Comment:
Could we do the same validation with https://api.flutter.dev/flutter/foundation/TargetPlatform-class.html?

Reply:
We can use a set